### DISCUSSION TITLE
Decoding "linked_action" codes

### DISCUSSION THREAD

**Discussion by Badkempachi** [glpi-project/glpi-agent#301] [Q&A]
Good afternoon,
Can you please tell me where can I find information with the decryption of the “linked_action” codes located in the “glpi_logs” table?

Question number 2.
![sql](https://user-images.githubusercontent.com/97535165/207796011-ad4121a7-5d56-488b-8a89-058a60afcdd2.jpg)
I also can’t understand the appearance of the “Item_SoftwareVersion” events; I don’t see information about changes in the history of the computer. Is that how it should be?
![history](https://user-images.githubusercontent.com/97535165/207796108-6d97a389-5972-4104-b482-f30ba069e9b0.jpg)

I created a topic on the forum about this, but it remained unanswered
https://forum.glpi-project.org/viewtopic.php?pid=497567#p497567

**Comment by g-bougard** [comment]
Hi @Badkempachi 

you're talking about GLPI code, not GLPI-Agent one. So your answer will be in [GLPI Project code](https://github.com/glpi-project/glpi).
